### PR TITLE
qqbrowser 5.0.4.213 (new cask)

### PR DESCRIPTION
**Pull request by Loyalsoldier**
After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/q/qqbrowser.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  name "QQBrowser"
+  name "QQ浏览器"
+  desc "QQ web browser"
+  homepage "https://browser.qq.com/mac/"
+
+  # The appcast file is outdated and does not contain the latest version
+  # https://dldir1v6.qq.com/invc/tt/QB/Public/qbmac/appcast.xml
+  livecheck do
+    skip "No reliable way to get version info"
+  end
```
Comment (Aaron-212) [inline Casks/q/qqbrowser.rb:18]:
```suggestion
  livecheck do
    url :url
    strategy :extract_plist do |items|
      items["com.tencent.qqbrowserappmac"].short_version
    end
  end
```

<https://docs.brew.sh/Brew-Livecheck#extractplist-strategy-block>

**Review by miccal — DISMISSED** [review]

**Comment by krehel** [comment]
Please source an alternative livecheck if possible. Having auto_update means it is checking somewhere.

Which then leads to a versioned url and sha256.

Not approving in current state, sorry.

**Comment by Aaron-212** [comment]
QQ Browser is based on Chrome, and I found that google-chrome.rb uses extract_plist strategy.

**Comment by Loyalsoldier** [comment]
Don't bother. I close it. Actually I don't use QQ browser. LOL

**Pull request closed without merging**